src/app/personside/visittkort-v2/header/EgenAnsattFeilendeSystemModal.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { useState } from 'react';
+import RawModal from 'nav-frontend-modal';
+import KnappBase from 'nav-frontend-knapper';
+import styled from 'styled-components/macro';
+import { Systemtittel } from 'nav-frontend-typografi';
+import { ReactComponent as ErrorIkon } from '../../../../svg/alvorlig-advarsel.svg';
+
+const Modal = styled(RawModal)`
+    text-align: center;
+    min-width: 20rem;
+`;
+const Knapp = styled(KnappBase)`
+    width: 100%;
+`;
+
+interface Props {
+    egenAnsattFeiler: boolean;
+}
+
+function EgenAnsattFeilendeSystemModal(props: Props) {
+    const [open, setOpen] = useState(props.egenAnsattFeiler);
+
+    return (
+        <Modal
+            isOpen={open}
+            contentLabel="FeilendeSystemEgenAnsatt"
+            closeButton={false}
+            shouldCloseOnOverlayClick={false}
+            onRequestClose={() => {
+                setOpen(false);
+            }}
+        >
+            <ErrorIkon width="2rem" className="blokk-xs" />
+            <Systemtittel tag="h1" className="blokk-xs" role="alert" aria-live="assertive">
+                OBS!!
+            </Systemtittel>
+
+            <div className="noncenter typo-normal blokk-m">
+                <p>{'Feilet ved uthenting av informasjon om egen ansatt.'}</p>
```

Comment:
Kan erstattes med `<p>Feilet ved uthenting av informasjon om egen ansatt.</p>` bare for å gjøre det litt enklere å lese